server/src/fat/java/com/ibm/ws/lars/rest/PermissionTest.java

```diff
@@ -50,7 +50,7 @@
     @Rule
     public RepositoryContext adminContext;
 
-    @Rule
+    // This is not a rule, as a rule does setup/tear down which is not wanted.
```

Comment:
https://hc.apache.org/httpcomponents-client-ga/tutorial/html/fundamentals.html#d5e217 suggests it should be closed, at least if we're creating a new one for each test.